scripts/cmake/version.cmake

```diff
@@ -59,6 +59,27 @@ if(NOT SOF_TAG)
 	set(SOF_TAG 0)
 endif()
 
+# list tracked files from src directory
+execute_process(COMMAND git ls-files src/*
```

Comment:
is this any different from just `git ls-files stc`?

Reply:
I didn't notice any difference in results, I used `src/*` to indicate that I want files from some directory. Updated to version with `src`.